adjust search path for dictionaries to conform with the standard
according to https://dict-common.alioth.debian.org/dsdt-policy.html#installdir
---
 CMakeLists.txt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- texworks-debian.git.orig/CMakeLists.txt
+++ texworks-debian.git/CMakeLists.txt
@@ -165,7 +165,7 @@
 
 IF (UNIX AND NOT APPLE)
   SET(TeXworks_HELP_DIR share/doc/texworks-help CACHE PATH "Path where TeXworks looks for help files")
-  SET(TeXworks_DIC_DIR /usr/share/myspell/dicts CACHE PATH "Path where TeXworks looks for spellchecking dictionaries")
+  SET(TeXworks_DIC_DIR /usr/share/hunspell CACHE PATH "Path where TeXworks looks for spellchecking dictionaries")
   SET(TeXworks_DOCS_DIR share/doc/texworks CACHE PATH "Path in which documentation files are installed")
   SET(TeXworks_ICON_DIR share/pixmaps CACHE PATH "Path in which icons are installed")
   SET(TeXworks_MAN_DIR share/man/man1 CACHE PATH "Path in which manpages are installed")
